tsconfig.json

```diff
@@ -26,7 +26,7 @@
         "experimentalDecorators": true, // Enables experimental support for ES decorators
         "incremental": true, // Enable incremental compilation by reading/writing information from prior compilations to a file on disk
         "noFallthroughCasesInSwitch": true, // Report errors for fallthrough cases in switch statement
-        "lib": ["dom"]
+        "lib": ["dom", "es2019.array"]
```

Comment:
Needed this to get access to `Array.flatMap` - any reason not to enable this? cc @mariusandra 

https://caniuse.com/?search=array.flatmap

Reply:
We actually had someone use a derpy browser before, which didn't support [Object.entries](https://caniuse.com/?search=object.entries), prompting the addition [of this](
https://github.com/PostHog/posthog/blob/43816096d0ecafc2a9370d976f806c8a90835699/frontend/src/index.html#L9) to index.html

Let's add flatMap there as well for now?